content/en/dora_metrics/setup/deployments.md

```diff
@@ -0,0 +1,216 @@
+```shell
+  curl -X POST "https://api.<DD_SITE>/api/v2/dora/deployment" \
+  -H "Accept: application/json" \
+  -H "Content-Type: application/json" \
+  -H "DD-API-KEY: ${DD_API_KEY}" \
+  -d @- << EOF
+  {
+    "data": {
+      "attributes": {
+        "service": "shopist",
+        "started_at": 1693491974000000000,
+        "finished_at": 1693491984000000000,
+        "git": {
+          "commit_sha": "66adc9350f2cc9b250b69abddab733dd55e1a588",
+          "repository_url": "https://github.com/organization/example-repository"
+        },
+        "env": "prod"
+      }
+    }
+  }
+EOF
+```
+
+[1]: /api/latest/dora-metrics/#send-a-deployment-event-for-dora-metrics
+[2]: https://app.datadoghq.com/organization-settings/api-keys
+{{% /tab %}}
+
+{{% tab "datadog-ci CLI" %}}
+
+The [`datadog-ci`][1] CLI tool provides a shortcut to send deployment events within your Continuous Integration environment.
+
+For the following example, set the `DD_SITE` environment variable to {{< region-param key="dd_site" code="true" >}} and set the `DD_API_KEY` environment variable to your [Datadog API Key][2]:
+```shell
+export DD_BETA_COMMANDS_ENABLED=1
+export DD_SITE="<DD_SITE>"
+export DD_API_KEY="<API_KEY>"
+
+export deploy_start=`date +%s`
+./your-deploy-script.sh
+datadog-ci dora deployment --service shopist --env prod \
+    --started-at $deploy_start --finished-at `date +%s` \
+    --git-repository-url "https://github.com/organization/example-repository" \
+    --git-commit-sha 66adc9350f2cc9b250b69abddab733dd55e1a588
+```
+
+The deployment finish time is automatically set to now if `--finished-at` is not provided.
+
+If the deployment CI job is running on the exact same Git revision that is being deployed, `git-repository-url` and `git-commit-sha` can be omitted and will be automatically inferred from the CI context.
+The `--skip-git` option can be provided to disable sending the repository URL and commit SHA. When this option is added, the Change Lead Time metric will not be available.
+
+[1]: https://www.npmjs.com/package/@datadog/datadog-ci
+[2]: https://app.datadoghq.com/organization-settings/api-keys
+{{% /tab %}}
+{{< /tabs >}}
+
+## Calculating Change Lead Time
+
+For a single Git commit, the Change Lead Time (CLT) is calculated as time from the creation of the commit to when the deployment including that commit was executed.
+To calculate the Change Lead Time for a deployment, Datadog performs the [Git Log][6] between the deployment commit SHA and the previous deployment commit SHA to find all the commits being deployed. Then, it computes the average of the related Change Lead Time values.
+Change Lead Time is not available for the first deployment of a service that includes Git information. Datadog doesn't store the actual content of files in your repository, only Git commit and tree objects.
```

Comment:
Remove `Change Lead Time is not available for the first deployment of a service that includes Git information.` since it's already in the Limitations section

Reply:
Ah, good spot, done!